sailpoint.sgnl.yaml

```diff
@@ -0,0 +1,1575 @@
+icon: PHN2ZyB3aWR0aD0iNjQiIGhlaWdodD0iNjQiIHZpZXdCb3g9IjAgMCA2NCA2NCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZD0iTTQwLjQxMTggNDIuMTM3M0gxMEwzNS4yMzUzIDEwTDQwLjQxMTggNDIuMTM3M1oiIGZpbGw9IiMwMDMzQTEiLz4KPHBhdGggZD0iTTQwLjQxMTggNDIuMTM3M0g1NEwzNS4yMzUzIDEwTDQwLjQxMTggNDIuMTM3M1oiIGZpbGw9IiNDQzI3QjAiLz4KPHBhdGggZD0iTTQwLjQxMTggNDIuMTM3M0gxMEw0Mi4zNTI5IDUzLjc4NDNMNDAuNDExOCA0Mi4xMzczWiIgZmlsbD0iIzAwNzFDRSIvPgo8cGF0aCBkPSJNNDAuNDExOCA0Mi4xMzczSDU0TDQyLjM1MjkgNTMuNzg0M0w0MC40MTE4IDQyLjEzNzNaIiBmaWxsPSIjRTE3RkQyIi8+Cjwvc3ZnPgo=
+description: "Sailpoint as a System of Record"
+address: "{{Input Required: Sailpoint URL}}"
+defaultSyncFrequency: HOURLY
+defaultSyncMinInterval: 1
+defaultApiCallFrequency: SECONDLY
+defaultApiCallMinInterval: 1
+type: "Sailpoint2.0-1.0.0"
+# Example Configs:
+# empty config: {} | b64: "e30="
+# config with filtering on Users: {"Users":{"queryParams":{"filter":"displayName eq \"SGNL\""}}} | b64: "eyJVc2VycyI6eyJxdWVyeVBhcmFtcyI6eyJmaWx0ZXIiOiJkaXNwbGF5TmFtZSBlcSBcIlNHTkxcIiJ9fQ=="
+# config with filter, ordering and sortBy on Groups {"Groups":{"queryParams":{"filter":"displayName eq \"SGNL\"","sortBy":"userName","ascending":true}}} | b64: "eyJHcm91cHMiOnsicXVlcnlQYXJhbXMiOnsiZmlsdGVyIjoiZGlzcGxheU5hbWUgZXEgXCJTR05MXCIiLCJzb3J0QnkiOiJ1c2VyTmFtZSIsImFzY2VuZGluZyI6dHJ1ZX19fQ=="
+adapterConfig: "e30=" # {}
+# This must match the list of supportedAuthMechanisms specified on the Adapter.
+auth:
+  - basic:
+      username: "{{Input Required: Username}}"
+      password: "{{Input Required: API Token}}"
+entities:
+  Account:
+    displayName: SailpointAccount
+    externalId: Accounts
+    description: Entity representing a Sailpoint Identity IQ Account.
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: nativeIdentity
+        externalId: nativeIdentity
+        type: String
+      - name: displayName
+        externalId: displayName
+        type: String
+      - name: instance
+        externalId: instance
+        type: String
+      - name: uuid
+        externalId: uuid
+        type: String
+      - name: active
+        externalId: active
+        type: Bool
+      - name: locked
+        externalId: locked
+        type: Bool
+      - name: lastRefresh
+        externalId: lastRefresh
+        type: DateTime
+      - name: lastTargetAggregation
+        externalId: lastTargetAggregation
+        type: DateTime
+      - name: manuallyCorrelated
+        externalId: manuallyCorrelated
+        type: Bool
+      - name: hasEntitlements
+        externalId: hasEntitlements
+        type: Bool
+      - name: identityValue
+        externalId: identity.value
+        type: String
+      - name: identityRef
+        externalId: identity.["$ref"]
```

Comment:
Have you verified this will work? Are you trying to match a key on the `identity` object named `$ref`? If so, shouldn't that just be `$.identity.$ref`? Same for the rest of the file

Reply:
The documentation actually just looks to have `ref` listed as any other field, shouldn't this just be `$.identity.ref`?

https://developer.sailpoint.com/iiq/api/get-accounts